src/twisted/newsfragments/9384.bugfix

```diff
@@ -0,0 +1,3 @@
+Syntax error under Python 3.7 fixed for twisted.conch.manhole and
+twisted.main.imap4. isAsync keyword argument is used by default instead of
+async, async keywoard argument is deprecated.
```

Comment:
The deprecation info should be put into a separate file named src/twisted/newsfragments/9384.removal

In this way, the depreciation part of the release notes will contain the info about `async` keyword.
Otherwise, the bugfix part will contain both bugfix and deprecation.